### ISSUE TITLE
Remove WeaveWorks URLs from Kubernetes Documentation Website

### ISSUE THREAD

**Issue by seifrajhi** [kubernetes/website#45482]
As per the announcement by Richardson Alexis on LinkedIn ([link](https://www.linkedin.com/posts/richardsonalexis_hi-everyone-i-am-very-sad-to-announce-activity-7160295096825860096-ZS67)), WeaveWorks is winding down its operations. Consequently, many URLs from WeaveWorks in the Kubernetes documentation are now obsolete, leading to a "Site Not Found" error message.

For instance [here](https://github.com/kubernetes/website/blob/main/content/en/case-studies/ocado/index.html#L54)

Proposal:
1. Identify all WeaveWorks URLs in the documentation.
2. Remove or replace these URLs with alternative resources to ensure the documentation remains accurate and useful.
3. Consider updating documentation guidelines to periodically review and update third-party URLs.

This action will ensure users aren't directed to broken or inaccessible resources.

**Comment by stmcginnis** [comment]
I don't think we want to remove this just because the company originally behind it is out of business. It is an open source project that at [least appears to be living on.](https://github.com/kubernetes/website/pull/45445)

**Comment by seifrajhi** [comment]
Hello @stmcginnis ,
Yes, as I proposed  in my issue we can `Remove or replace these URLs with alternative resources to ensure the documentation remains accurate and useful.`

Because their website  (https://www.weave.works/) is officially decommissioned and not accessible anymore.

**Comment by a-mccarthy** [comment]
So this seems to be coming up a bunch of other issues as well, https://github.com/kubernetes/website/issues/45201 and a few other issues/prs. So i think its worth talking about to get on the same page about what to do about the links. 

> Consider updating documentation guidelines to periodically review and update third-party URLs.

I'm not sure if there was ever a conversation around this when we put together the [third party content policy](https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content). I think we generally rely on the community to help keep the links updated, as we might not have the depth of knowledge or dedicated resources to always staff doing this research with in the SIG.

**Comment by sftim** [comment]
Also see #47421

/remove-lifecycle rotten

**Comment by utkarsh-singh1** [comment]
Seems like A need to create umbrella issue.

**Comment by utkarsh-singh1** [comment]
WeaveWorks github profile is also archived on 20 June 2024.

**Comment by utkarsh-singh1** [comment]
It would be better to remove links and tutorials related to WaveWorks.

**Comment by utkarsh-singh1** [comment]
https://github.com/weaveworks/weave

**Comment by sftim** [comment]
Also see https://github.com/kubernetes/website/issues/47441
